eth/db/chain.py

```diff
@@ -444,12 +449,81 @@ def _add_transaction_to_canonical_chain(db: DatabaseAPI,
         - add lookup from transaction hash to the block number and index that the body is stored at
         - remove transaction hash to body lookup in the pending pool
         """
-        transaction_key = TransactionKey(block_header.block_number, index)
+        transaction_key = BlockDataKey(block_header.block_number, index)
         db.set(
             SchemaV1.make_transaction_hash_to_block_lookup_key(transaction_hash),
             rlp.encode(transaction_key),
         )
 
+    #
+    # Withdrawals API
+    #
+
+    def persist_withdrawals(
```

Comment:
**Note for refactoring:** This wasn't a very useful approach and I didn't use it... can remove these.